modules/test/playwright/tests/osb-faro-web/page.spec.ts

```diff
@@ -64,6 +64,185 @@ const goToWithReferrer = async function (page, referrer, url) {
+				},
+			],
+			firstName: 'user1',
+			id: 'user1@liferay.com',
+			lastName: 'user1',
+		},
+		{
+			emailAddress: 'user1@liferay.com',
+			fields: [
+				{
+					dataSourceId: 0,
+					name: 'givenName',
+					value: 'user2',
+				},
+				{
+					dataSourceId: 0,
+					name: 'familyName',
+					value: 'user2',
+				},
+				{
+					dataSourceId: 0,
+					name: 'email',
+					value: 'user2@liferay.com',
+				},
+			],
+			firstName: 'user2',
+			id: 'user2@liferay.com',
+			lastName: 'user2',
+		},
+		{
+			emailAddress: 'user3@liferay.com',
+			fields: [
+				{
+					dataSourceId: 0,
+					name: 'givenName',
+					value: 'user3',
+				},
+				{
+					dataSourceId: 0,
+					name: 'familyName',
+					value: 'user3',
+				},
+				{
+					dataSourceId: 0,
+					name: 'email',
+					value: 'user3@liferay.com',
+				},
+			],
+			firstName: 'user3',
+			id: 'user3@liferay.com',
+			lastName: 'user3',
+		},
+	]);
+
+	const date1 = new Date();
+
+	await apiHelpers.jsonWebServicesOSBAsah.createIdentities([
+		{
+			createDate: date1.toISOString(),
+			id: '1',
+			individualId: 'user1@liferay.com',
+		},
+		{
+			createDate: date1.toISOString(),
+			id: '2',
+			individualId: 'user2@liferay.com',
+		},
+		{
+			createDate: date1.toISOString(),
+			id: '3',
+			individualId: 'user3@liferay.com',
+		},
+	]);
+
+	await apiHelpers.jsonWebServicesOSBAsah.createEvents([
+		{
+			applicationId: 'Page',
+			canonicalUrl: 'https://www.liferay.com',
+			channelId: channel.id,
+			eventDate: date1.toISOString(),
+			eventId: 'pageViewed',
+			title: 'Liferay',
+			userId: '1',
+		},
+		{
+			applicationId: 'Page',
+			canonicalUrl: 'https://www.liferay.com',
+			channelId: channel.id,
+			eventDate: date1.toISOString(),
+			eventId: 'pageViewed',
+			title: 'Liferay',
+			userId: '2',
+		},
+	]);
+
+	const date2 = new Date();
+
+	date2.setDate(date2.getDate() - 5);
+
+	await apiHelpers.jsonWebServicesOSBAsah.createPagesDaily([
+		{
+			canonicalUrl: 'https://www.liferay.com',
+			channelId: channel.id,
+			eventDate: date2.toISOString(),
+			title: 'Liferay',
+			userId: '3',
+			views: 1,
+		},
+	]);
+
+	await page.goto(
+		`${faroConfig.environment.baseUrl}/workspace/${project.groupId}/${channel.id}/sites`
+	);
+
+	await page.getByRole('link', {exact: true, name: 'Pages'}).click();
+
+	await page.getByRole('link', {name: 'Liferay'}).click();
+
+	await page.getByRole('link', {name: 'Known Individuals'}).click();
+
+	await expect(
+		page.getByRole('cell', {name: 'user3 user3 user3@liferay.com'})
+	).toBeVisible({
+		timeout: 100 * 1000,
+	});
+
+	await page.getByRole('button', {name: 'Last 30 days'}).click();
+
+	await page.getByRole('menuitem', {name: 'Last 24 hours'}).click();
+
+	await expect(
+		page.getByRole('cell', {name: 'user1 user1 user1@liferay.com'})
+	).toBeVisible({
+		timeout: 100 * 1000,
+	});
+
+	await expect(
+		page.getByRole('cell', {name: 'user2 user2 user1@liferay.com'})
+	).toBeVisible({
+		timeout: 100 * 1000,
+	});
```

Comment:
Hey @marcosapmf, I think it would be good to also check that `user3` is not appearing in the list since you changed it to 24h